extensions/arc/src/localizedConstants.ts

```diff
@@ -150,9 +151,11 @@ export const computeAndStorageDescriptionPartSix = localize('arc.computeAndStora
 export const node = localize('arc.node', "node");
 export const nodes = localize('arc.nodes', "nodes");
 export const workerNodes = localize('arc.workerNodes', "Worker Nodes");
+export const coordinatorNode = localize('arc.coordinatorNode', "Coordinator Node");
 export const storagePerNode = localize('arc.storagePerNode', "storage per node");
 export const workerNodeCount = localize('arc.workerNodeCount', "Worker node count:");
 export const configurationPerNode = localize('arc.configurationPerNode', "Configuration (per node)");
+export const configurationCoordinatorNode = localize('arc.configurationCoordinatorNode', "Configuration");
```

Comment:
nit: just name this `arc.configuration`. You can reuse strings and it makes it easier to find and re-use if you keep it generic

Reply:
meant to do that haha